### PR TITLE
Fs 2277 feature toggle spike

### PR DESCRIPTION
**Pull request by Adam-W1**
FS-2277: Spike: Feature Toggles

### Change description
changed the name of authenticator_data to redis_data as this redis service is going to be used in other repos as well. also added a redis_uri envvar under the assessment service as we are testing the toggling spike using this repo to begin with.

### How to test
checkout to this branch along with [this ](https://github.com/communitiesuk/funding-service-design-utils/pull/62)in utils and [this ](https://github.com/communitiesuk/funding-service-design-assessment/pull/191)in assessment and see flagging functionality hidden from the user. (More info on running this E2E can be found in the description of the above assessment PR^.

See [this doc](https://digital.dclg.gov.uk/confluence/display/FS/Feature+Toggles+in+FSD) for more information on methodology, set up and how to use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Copy `.env.example` to `.env` and ask another team member for the missing secret value
 * `docker compose up`
 * Apps should be running on localhost on the ports in the [docker-compose.yml](docker-compose.yml) `ports` key before the `:`
+* Note: When testing locally using the docker runner, docker might use the cached version of fsd_utils (or any another depedency). To avoid this and pick up your intended changes, run `docker compose build <service_name> --no-cache` first before running `docker compose up` or just running `docker compose up --build` on its own.
```
Comment (ramsharma-prog) [inline README.md:11]: does this command works? `docker compose up --build`. I have been using `docker-compose up --build` but it didn't seem to clear the cache

Comment (ramsharma-prog) [reply README.md:11]: `docker-compose up --build --force-recreate --no-cache`

Comment (gio-karunakaran) [reply README.md]: updated this paragraph to avoid confusion 👍🏾

**Review by ramsharma-prog — APPROVED** [review]
LGTM

**Pull request merged**